proto/serverless/instrumentation/v1/dev_mode.proto

```diff
@@ -17,10 +17,24 @@ message DevModePayload {
   // The lambda request id where this payload originated from
   string request_id = 3;
 
+  // Extracted Lambda Telemetry API data
+  optional LambdaTelemetry telemetry = 6;
+
   oneof payload {
     // The set of lambda traces that were generated via an internal extension
     TracePayload trace = 4;
     // The req or response data from the instrumented lambda function
     RequestResponse request_response = 5;
   }
 }
+
+// Lambda Telemetry API data. This data is only available for lambda functions that
+// have access to the telemetry API so it will not be included in all regions.
+message LambdaTelemetry {
+  // Init duration as reported by the metrics on the platform.initReport event
+  optional uint32 init_duration = 1;
+  // Internal runtime duration as reported by the metrics on the platform.runtimeDone event
+  optional uint32 internal_duration = 2;
```

Comment:
Is this reported by new telemetry API? (At least I don't see such duration reported in current version)

Reply:
Now that you are asking I am questioning it but I thought that is what the `metrics.durationMs` was supposed to be 🤔 

<img width="841" alt="Screen Shot 2022-10-10 at 3 53 19 PM" src="https://user-images.githubusercontent.com/1753824/194951030-70667b6c-c5da-4150-be46-25b469680e6d.png">

Reply:
Ok, I take it's available only with new telemetry API, at least my records of `platform.runtimeDone` look as:

```json
{
	"time": "2022-05-04T13:29:32.659Z",
	"type": "platform.runtimeDone",
	"record": {
		"requestId": "5f48a34d-ab8e-455c-aabd-818bc6fe1d31",
		"status": "success"
	}
}
```

Reply:
Thanks for clarification 👍